node/modules/lp2p/addrs.go

```diff
@@ -11,19 +11,6 @@ import (
 	mamask "github.com/whyrusleeping/multiaddr-filter"
 )
 
-func AddrFilters(filters []string) func() (opts Libp2pOpts, err error) {
```

Comment:
Was this an unused function? Otherwise, you might need to replace it with a connection gater.

Reply:
Yeah, it was dead code.